src/pages/index.js

```diff
@@ -3,16 +3,40 @@ import FormValidator from "../components/FormValidator.js";
 import Section from "../components/Section.js";
 import ModalWithImage from "../components/ModalWithImage.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../utils/Api.js";
 import ModalWithForm from "../components/ModalWithForm.js";
-import { initialCards, config } from "../utils/constants.js";
+import { config } from "../utils/constants.js";
 
 import "./index.css";
+import ModalWithConfirm from "../components/ModalWithConfirm.js";
 
 /*--------------------------------------------------------------------------------------------------------------------------------------------------------------------------*/
 /*                                                                                                                                                                          */
 /*                                                                     Elements                                                                                                                                                 */
 /*                                                                                                                                                                          */
 /*--------------------------------------------------------------------------------------------------------------------------------------------------------------------------*/
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authToken: "d88a2b9a-3c17-46af-b0b4-5a460d3316a6",
+});
+
+let cardSection;
+
+api.getCards().then((initialCards) => {
+  console.log(initialCards);
+  cardSection = new Section(
+    {
+      items: initialCards,
+      renderer: (cardData) => {
+        const cardElement = createCard(cardData);
+        cardSection.addItem(cardElement);
+      },
+    },
+    ".cards__list"
+  );
+  cardSection.renderItems();
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**